app/models/public_body_category.rb

```diff
@@ -56,7 +56,8 @@ def self.without_headings
   end
 
   def default_locale?
-    locale == I18n.default_locale
+    return false if locale.nil?
+    AlaveteliLocalization.default_locale?(locale)
```

Comment:
handling it explicitly as `false` in case locale is ever purposely set to `""` as that will eval to `true` which is probably not what we want